packages/back-end/src/controllers/features.ts

```diff
@@ -723,6 +801,7 @@ export async function postFeatureFork(
     baseVersion: revision.version,
     changes: revision,
     environments,
+    requiresReview: !!org.settings?.requireReviews && !!req.superAdmin,
```

Comment:
Don't check `superAdmin`.  That's a special setting only for multi-org deployments and not relevant here.